modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -71,13 +71,12 @@ Available resources vary between clusters. The number of possible clusters withi
 [id="installation-vsphere-installer-infra-requirements-networking_{context}"]
 == Networking requirements
 
-You must use DHCP for the network and ensure that the DHCP server is configured to provide persistent IP addresses and host names to the cluster machines.
-Additionally, you must create the following networking resources before you install the {product-title} cluster:
+You must use DHCP for the network and ensure that the DHCP server is configured to provide persistent IP addresses and host names to the cluster machines. Additionally, you must create the following networking resources before you install the {product-title} cluster:
```

Comment:
Do we need hostnames?

Reply:
@patrickdillon Should we remove host names here?

Reply:
AFAIK in 4.6 hostnames for ETCD aren't required anymore.

Reply:
Thanks, I removed host names.